pyproject.toml

```diff
@@ -70,7 +70,7 @@ warn_return_any = true
 plugins = "pydantic.mypy"
 
 [[tool.mypy.overrides]]
-module = ["ruamel", "bugzilla", "atlassian", "atlassian.rest_client", "statsd.defaults.env"]
+module = ["ruamel", "bugzilla", "atlassian", "atlassian.rest_client", "statsd.defaults.env", "dockerflow"]
```

Comment:
Are we missing a poetry.lock change?

Reply:
I don't think `mypy` config ends up in the lock file